src/mutable_data.rs

```diff
@@ -714,7 +714,7 @@ impl Data {
         }
     }
 
-    pub fn owner(&self) -> PublicKey {
+    pub fn owners(&self) -> PublicKey {
```

Comment:
This should stay as `owner` because generally we shouldn't care if it's a multisig/BLS key or some other key.
It'd be better to change the `.owners` field name in `SeqMutableData` and `UnseqMutableData`.